### DISCUSSION TITLE
Error which running dev container

### DISCUSSION THREAD

**Discussion by amankitsingh** [forem/forem#20249] [Q&A] [answered]
Hey, 
I am facing the below issue while running `Dev Containers: Rebuild and Reopen in Container`.
Using Mac M1.

It states some file missing issue

<img width="1728" alt="Screenshot 2023-10-16 at 23 45 13" src="https://github.com/forem/forem/assets/26693197/c3654bce-801b-4296-bff8-bdae431f45f5">

**Comment by maestromac** [comment]
Did you make changes to directory name? I think that may have been the reason. Try it again without altering folder name or structure

**Reply by amankitsingh** [reply, marked as answer]
Yes, I did change the folder name. 
I renamed it to the original folder name(forem) and it started working.

thankyou
